src/ContentClient.ts

```diff
@@ -403,7 +420,14 @@ export type DeploymentOptions<T> = {
   filters: DeploymentFilters
   sortBy?: DeploymentSorting
   fields?: DeploymentFields<T>
+  /**
+   * @deprecated please use try-catch and iterators instead
+   */
   errorListener?: (errorMessage: string) => void
+  /**
+   * Amount of elements per page
+   */
+  limit?: number
```

Comment:
isn't `limit` already in `DeploymentFilters`? If not, it should be there (in catalyst-commons)